docs/io.cozy.contacts.md

```diff
@@ -40,6 +41,9 @@ The `io.cozy.contacts` doctype is loosely based on the [vCard RFC](https://tools
     - `apartment?`: {string} Apartment number
     - `entrycode?`: {string} Entry code
   - `formattedAddress?`: {string} Unstructured version of the address
+  - `geo?`: {object}
+    - `geo?`: {array} Coordinates of the address, must be `[long, lat]`
+    - `cozyCategory?`: {"home|work"} The category of the address type
```

Comment:
This will probably evolve soon, I would use `home, work` as example rather than a fixed enum

Reply:
évoluer comment ? On peut changer l'enum par la suite non ? Si ça évolue il faudra changer la doc non ? Autant être le plus explicite et complet possible dans la doc à mon avis... ?

Reply:
On peut changer, mais comme on veut le faire évoluer probablement rapidement pour supporter d'autres motif, e.g. "sport", "school", etc, autant mettre des valeurs possibles en example, ce qui ne fait perdre aucune info, sans avoir forcément besoin de mettre à jour à chaque fois

Reply:
Perso je préfère qu'une doc soit exhaustive, à laquelle on peut se référer pour avoir tous les cas possibles. Nous verrons bien comment cela évolue 👍

Reply:
On le préfère tous, mais on sait aussi que c'est compliqué ;) 
Ca me fait d'ailleurs penser qu'à terme il faudra sans doute déplacer ces catégories dans cozy-client. Mais on verra ça le moment venu

Reply:
On le préfère tous, mais on sait aussi que c'est compliqué ;) 
Ca me fait d'ailleurs penser qu'à terme il faudra sans doute déplacer ces catégories dans cozy-client. Mais on verra ça le moment venu